src/app/core/config/config-fix.ts

```diff
@@ -1301,8 +1300,8 @@ export const defaultJsonConfig = {
         anonymize: "retain",
       },
       result: {
-        dataType: "number",
-        label: $localize`:Label for the percentage result of a relation:Result`,
+        dataType: "percentage",
+        label: $localize`:Label for the percentage result of a relation:Percentage`,
```

Comment:
```suggestion
        label: $localize`:Label for the percentage result of a relation:Result`,
```
This is the specific school year result for this specific entity type, different from the general datatype's label. Let's keep this as is.